utils/dbutils.py

```diff
@@ -0,0 +1,151 @@
+
+        ['vfatN','vfat3_ser_num', 'vfat3_barcode', 'iref', 'adc0m', 'adc1m', 'adc0b', 'adc1b', 'cal_dacm', 'cal_dacb']
+
+    vfatList    - list of VFAT Chip ID's.
+    fromProd    - If True (False) the view is taken from the Production (Development) DB
+    debug       - Prints additional info if true
+    """
+
+    df_vfatCalInfo = getVFAT3ProdSumView(vfatList, fromProd, debug)
+
+    return df_vfatCalInfo[['vfatN','vfat3_ser_num', 'vfat3_barcode', 'iref', 'adc0m', 'adc1m', 'adc0b', 'adc1b', 'cal_dacm', 'cal_dacb']]
+
+def getVFAT3ConfView(vfatList, fromProd=True, debug=False):
+    """
+    Gets the GEM_VFAT3_CHIP_CONF_V_RH view in the GEM DB for a list of input VFATs.
+
+    Returns a pandas dataframe object storing the data read from the DB
+
+    vfatList    - list of VFAT Chip ID's.
+    fromProd    - If True (False) the view is taken from the Production (Development) DB
+    debug       - Prints additional info if true
+    """
+
+    return getGemDBView("GEM_VFAT3_CHIP_CONF_V_RH",vfatList,fromProd,debug)
+
+def getVFAT3ProdSumView(vfatList, fromProd=True, debug=False):
+    """
+    Gets the GEM_VFAT3_PROD_SUMMARY_V_RH view in the GEM DB for a list of input VFATs.
+
+    Returns a pandas dataframe object storing the data read from the DB
+
+    vfatList    - list of VFAT Chip ID's.
+    fromProd    - If True (False) the view is taken from the Production (Development) DB
+    debug       - Prints additional info if true
+    """
+
+    return getGemDBView("GEM_VFAT3_PROD_SUMMARY_V_RH",vfatList,fromProd,debug)
+
+def getVFATFilter(vfatList):
+    """
+    Returns a string that can be used as a filter in an SQL query.
+
+    vfatList - list of VFAT Chip ID's
+    """
+
+    strRetFilter = " WHERE ("
+    for idx, vfatID in enumerate(vfatList):
+        if idx == 0:
+            strRetFilter += " data.VFAT3_SER_NUM='{0}'".format(str(hex(vfatID)).strip('L'))
+        else:
+            strRetFilter += " OR data.VFAT3_SER_NUM='{0}'".format(str(hex(vfatID)).strip('L'))
+            pass
+        pass
+    strRetFilter += " )\n"
+
+    return strRetFilter
+
+def joinOnVFATSerNum(vfatList, dfGemView):
+    """
+    Creates a dataframe object from vfatList with keys 'vfat3_ser_num' and 'vfatN'.
+    Then it joins this dataframe with dfGemView using the 'vfat3_ser_num'.
+
+    vfatList - A list of vfat ChipID's ordered by vfat position (sw)
+    dfGemView - A pandas dataframe containing the column name 'vfat3_ser_num'
+    """
+
+    if 'vfat3_ser_num' in dfGemView.columns:
+        dfVFATPos = pd.DataFrame(
+                    {   'vfatN':[vfat for vfat in range(24)], 
+                        'vfat3_ser_num':[str(hex(id)).strip('L') for id in vfatList]}
+                )
+
+        dfGemView = pd.merge(dfVFATPos, dfGemView, on='vfat3_ser_num', how='outer')
+    else:
+        printYellow("column 'vfat3_ser_num' not in input dataframe columns: {0}".format(dfGemView.columns))
+        pass
+
+    return dfGemView
```

Comment:
Standard comment about function/variable naming conventions (or use the same convention you adopted earlier... `df_gemView`)